modules/telestion-utils/src/main/java/de/wuespace/telestion/utils/FileUtils.java

```diff
@@ -0,0 +1,31 @@
+package de.wuespace.telestion.utils;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.stream.Collectors;
+
+/**
+ * Utilities for file handling in Java.
+ *
+ * @author Ludwig Richter (@fussel178)
+ * @see java.nio.file
+ */
+public class FileUtils {
+
+	/**
+	 * Reads the contents from a file.
+	 *
+	 * @param path the path to the file
+	 * @return the contents
+	 * @throws IOException if an I/O error occurs during opening the file
+	 */
+	public static String readFile(Path path) throws IOException {
+		var lines = Files.lines(path);
+
+		var content = lines.collect(Collectors.joining());
+		lines.close();
+
```

Comment:
There exists `Files.readString()` since Java 11 which reads an entire file into a string in Java.
This static Method is part of the `java.nio.file.Files` package. There also is a similar method for writing in one line: `Files.writeString`. Therefore I think this entire Utility class is sadly not really necessary.

Reply:
Ohh, good to know!

Reply:
Then no utils package :sweat_smile:

Reply:
Removed associated commits.